test/HttpServer.test.ts

```diff
@@ -0,0 +1,112 @@
+import { stub, createStubInstance, SinonStubbedInstance } from 'sinon';
+import { use, assert, request, expect } from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import chaiHttp from 'chai-http';
+import {
+    RelayServer,
+    HttpServer,
+    ServerDependencies,
+    TxStoreManager,
+    KeyManager,
+    ServerConfigParams
+} from '../src';
+import { ContractInteractor } from '@rsksmart/rif-relay-common';
+
+use(chaiAsPromised);
+use(chaiHttp);
+
+describe('HttpServer', () => {
+    const fakeServerConfig: Partial<ServerConfigParams> = {
+        url: 'http://localhost:8090'
+    };
+    const port = 8095;
+    const gasPrice = 5;
+    const method: keyof RelayServer = 'getMinGasPrice';
```

Comment:
I think this is unnecessary and misleading. It suggests that we only need to test this one method. I'd just use it directly where you need it.

Reply:
updated